components/jobsBoard.js

```diff
@@ -0,0 +1,134 @@
+        </p>
+        <Link
+          href="jobs/rules-guidelines"
+          target="_blank"
+        >
+          Rules and Guidelines
+        </Link>
+        <br />
+        <Button
+          buttonClassName="custom-button green-btn px-5 my-4"
+          anchorClassName="text-decoration-none text-light"
+          buttonLabel="Submit a Job"
+          buttonHyperLink="https://forms.gle/1b1VeRprtCUMcdJs9"
+        />
+      </div>
+      <div className="row my-4">
+        <div className="text-center">
+          {Array.isArray(jobs) && jobs.length === 0 &&
+            <h3
+              className="text-center"
+              data-aos="fade-down"
+              data-aos-duration="800"
+            >Currently No Job Listings Available – Check Back Soon!
+            </h3>
+          }
+        </div>
+        {jobs && jobs.map((job, index) => (
+          <div key={index} className="col-lg-4 col-md-6 col-12 mb-4">
+            <div className="jobs-card p-4">
+              {/* Start of Job Header */}
+              <div>
+                <h4>{job.jobTitle}</h4>
+                {job.companyWebsite ? <a href={job.companyWebsite} target="_blank">
+                  <div className="mb-1">
+                    <span className="me-2">
+                      <IconComponent name="building" size={30} />
+                    </span>
+                    {job.companyName}
+                  </div>
+                </a>
+                  :
+                  <div>
+                    <span className="me-2">
+                      <IconComponent name="building" size={30} />
+                    </span>
+                    {job.companyName}
+                  </div>
+                }
+                {job.jobLocation &&
+                  <div className="mb-1">
+                    <span className="me-2">
+                      <IconComponent name="location" size={30} />
+                    </span>
+                    {job.jobLocation}
+                  </div>
+                }
+                {job.jobType &&
+                  <div className="mb-1">
+                    <span className="me-2">
+                      <IconComponent name="briefCase" size={30} />
+                    </span>
+                    {job.jobType}
+                  </div>
+                }
+              </div>
+              {/* Start of Job Body */}
+              <div className="row mt-2">
+                {job.jobDescription &&
+                  <Paragraph
+                    text={job.jobDescription}
+                    isHtml={true}
+                    numberOfLines={3}
+                  />}
+                {job.skills &&
+                  <div className="mb-1">
+                    {"Skills: " + job.skills}
+                  </div>
```

Comment:
can we move skills up, before description? 
They seem to get lost between the stack of words.

Reply:
Sure

Reply:
This is incorporated.